src/SurveyDesign.jl

```diff
@@ -14,7 +13,9 @@ abstract type AbstractSurveyDesign end
 """
     SimpleRandomSample <: AbstractSurveyDesign
 
-Survey design sampled by simple random sampling.
+
+A simple random sample dataset can be loaded from a data frame into a `SimpleRandomSample` object for downstream analyses. 
```

Comment:
```suggestion
Survey design sampled by simple random sampling.
```

Reply:
“Survey design sampled by simple random sampling” is concise, but it doesn't make sense.  Similarly, the other ones, we can't accept them just because they are short.

Reply:
How about "Simple random sample survey design"?

Reply:
The fact that the information for a survey design comes from a data frame is clear from the signature of the constructor. What supporting sentences did you have in mind?

Reply:
It's clear to us, may not be clear to a new user.